flink-table/flink-table-common/src/main/java/org/apache/flink/table/functions/BuiltInFunctionDefinitions.java

```diff
@@ -629,7 +629,7 @@
                                             logical(LogicalTypeFamily.CHARACTER_STRING),
                                             logical(LogicalTypeFamily.CHARACTER_STRING),
                                             logical(LogicalTypeRoot.INTEGER))))
-                    .outputTypeStrategy(nullableIfArgs(explicit(DataTypes.STRING())))
+                    .outputTypeStrategy(forceNullable(explicit(DataTypes.STRING())))
```

Comment:
`forceNullable` is useful for a more "dynamic argument", but since this is static. you can simply express this as `explicit(DataTypes.STRING().nullable())`. Nullable is actually the default, but this would make it more explicit.